apps/website/src/utils/drupal-executor.ts

```diff
@@ -9,23 +9,52 @@ export function drupalExecutor(endpoint: string, forward: boolean = true) {
     variables?: OperationVariables<OperationId>,
   ) {
     const url = new URL(endpoint, window.location.origin);
-    url.searchParams.set('queryId', id);
-    url.searchParams.set('variables', JSON.stringify(variables));
-    const { data, errors } = await (
-      await fetch(url, {
-        credentials: 'include',
-        headers: forward
-          ? {
-              'SLB-Forwarded-Proto': window.location.protocol.slice(0, -1),
-              'SLB-Forwarded-Host': window.location.hostname,
-              'SLB-Forwarded-Port': window.location.port,
-            }
-          : {},
-      })
-    ).json();
-    if (errors) {
-      throw errors;
+    if (variables && variables.graphqlOperationType === 'mutation') {
```

Comment:
This feels like "additional knowledge" to me, that all other executors then would have to implement as well. I think we should derive from "OperationId". I think for now it would be good enough to search for `Mutation:` in `id`. And there is [SLB-259](https://amazeelabs.atlassian.net/browse/SLB-259) which could make this more solid in a later step.

Reply:
this should be fixed, the check uses the operation id now.